db/migrate/20230724203021_add_posts_count_to_users.rb

```diff
@@ -0,0 +1,11 @@
+class AddCounterCaches < ActiveRecord::Migration[7.0]
```

Comment:
- [ ] When I ran the migration I am getting this errors as you can see on the screenshot below 👇  that make impssible to run the Tests. I would like to recommend you to double-check the name of your migration  and make sure that all the tests are passed before submitting it as required for this project and previous reviewer 😄 

![unnamed (2)](https://github.com/joyapisi/blog_app/assets/6615394/a7da3a54-c936-46fb-a5d7-1f9cbe749549)